README.md

```diff
@@ -22,7 +22,7 @@ If a bot was available, it will now be running on uqcstesting.
 ## Running the bot in a custom Slack team
 
 1. [Create a Slack workspace](https://slack.com/create)
-2. Run `python3 setup.py install`
+2. Run `pip install -e .`
```

Comment:
this is my bad for forgetting to remove step 2 when I re-added it, but you can probably omit this line entirely seeing as though its stated in 'Setting up the bot'